common/transport/amqp/src/amqp.ts

```diff
@@ -408,9 +410,9 @@ export class Amqp {
    */
   setDisconnectHandler(disconnectCallback: GenericAmqpBaseCallback<any>): void {
     this._disconnectHandler = disconnectCallback;
-    this._amqp.on('connection:closed', () => {
-      this._disconnectHandler(new Error('amqp10: connection closed'));
-    });
+    // this._amqp.on('connection:closed', () => {
```

Comment:
>this._amqp.on('connection:closed', () => { [](start = 7, length = 42)

I'm guessing this is one of the issues you plan to fix, but could you please comment why you commented out in case you don't get to it or it becomes irrelevant.  (code that is commented out without explanation always scares me because I wonder why it wasn't removed and what terrible secret it's hiding.)

Reply:
yes, I want to fix this before we ship - and it's related to the deactivated e2e tests I believe.

---
In reply to: [145961633](https://github.com/Azure/azure-iot-sdk-node/pull/114#discussion_r145961633) [](ancestors = 145961633)